data_sources/config.py

```diff
@@ -17,6 +17,16 @@
     },
 ]
 
+"""
+Mapping AMANDA work zone types to WZDX work zone types
+"""
+
+work_zone_type_mapping = {
+    "mobile": "planned-moving-area",
+    "daily": "planned-moving-area",
+    "24/7": "static",
+}
```

Comment:
This is how we convert from AMANDA work zone types to WZDx types

definitions are here:
https://github.com/usdot-jpo-ode/wzdx/blob/develop/spec-content/enumerated-types/WorkZoneType.md

if none are supplied from AMANDA, it will default to `static`